app/views/errors/notFound.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@()(implicit appConfig: config.AppConfig, request: Request[_], messages: Messages)
+
+@views.html.errors.standardError(
+  appConfig,
+  messages("notFound.title"),
+  messages("notFound.heading"),
+  messages("notFound.message")
+)
```

Comment:
The messages for the Not found need adding to the messages file
<img width="1707" alt="screen shot 2018-05-31 at 10 49 25" src="https://user-images.githubusercontent.com/8091191/40777135-d5a0c7fe-64c4-11e8-9f89-ab3e7eb304c6.png">